components/textarea/textarea.tsx

```diff
@@ -4,6 +4,8 @@ import withDefaults from '../utils/with-defaults'
 import { NormalTypes } from '../utils/prop-types'
 import { getColors } from '../input/styles'
 
+type ResizeTypes = 'none' | 'both' | 'horizontal' | 'vertical' | 'initial' | 'inherit' 
```

Comment:
`ResizeTypes` should be a `tuple`:
```
const resizeTypes = tuple('none', ...)
type ResizeTypes = typeof resizeTypes[number]
```

Reply:
updated commit: https://github.com/geist-org/react/pull/416/commits/ec1ba50fb476e625f49f631a7c41426db247a8e0